src/emscripten-optimizer/parser.cpp

```diff
@@ -14,101 +14,103 @@
  * limitations under the License.
  */
 
+#include <unordered_map>
+
```

Comment:
This is now necessary because it used to be included (unnecessarily!) from src/emscripten-optimizer/istring.h, but the new istring.h does not include it. It would be nice if there were a good way to keep the codebase include-what-you-use clean, but last time I looked into it the tooling output wasn't stable enough.